.slackbot.yml

```diff
@@ -0,0 +1,15 @@
+releaseSchedule:
+  dbagnell: 2/4/2019
```

Comment:
Why are we skipping Fridays now?  We might skip outreach at some point on Fridays but I don't that we should make the release guideline more confusing here for unknown gain.

Reply:
I was just skipping weekends, so that was a mistake. Thanks for pointing it out!